libs/domain/product/title/src/title.component.ts

```diff
@@ -33,28 +31,64 @@ export class ProductTitleComponent extends ProductMixin(
   });
 
   protected override render(): TemplateResult | void {
-    const { tag, as, asLg, asMd, asSm, maxLines } = this.$options();
-
-    return html`<oryx-heading
-      .tag=${tag}
-      .maxLines=${maxLines}
-      .as=${as}
-      .asLg=${asLg}
-      .asMd=${asMd}
-      .asSm=${asSm}
-    >
-      ${this.$link() ? this.renderLink() : html`${this.$product()?.name}`}
-    </oryx-heading>`;
+    const name = this.$product()?.name;
+    if (!name) return;
+    if (featureVersion <= '1.4') {
```

Comment:
Shou;dn't it be `< '1.4'`? 
Also we decided to use approach with `>= '1.4'` instead.